packages/react-sdk/src/core/hooks/useDevices.ts

```diff
@@ -1,11 +1,50 @@
-import { useEffect, useState } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 import { Observable, pairwise } from 'rxjs';
 import {
   getAudioDevices,
   getAudioOutputDevices,
   getVideoDevices,
 } from '@stream-io/video-client';
 
+export const useCheckBrowserPermissions = (permissionName: PermissionName) => {
```

Comment:
I'd name this hook `useHasBrowserPermission`. "Check" feels like more "momentarily" assertion, while this hook can emit a new value anytime.

Reply:
Good idea, I have renamed it